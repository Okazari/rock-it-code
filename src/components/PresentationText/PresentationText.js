import React from "react"
import styled from "styled-components"

const TextWrapper = styled.div`
  text-align: center;
`

const Title = styled.h1`
  color: white;
  margin: 0;
  font-weight: lighter;
`

const Subtitle = styled.h2`
  color: white;
  font-weight: lighter;
  margin: 0;
`

const PresentationText = () => {
  return (
    <TextWrapper>
      <Title>Welcome to Rock it Code</Title>
      <Subtitle>Simply paste your code in the editor below, press play and let it rock</Subtitle>
    </TextWrapper>
  )
}

export default PresentationText
