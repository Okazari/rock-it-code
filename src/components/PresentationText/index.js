export { default as PresentationText } from "./PresentationText"
