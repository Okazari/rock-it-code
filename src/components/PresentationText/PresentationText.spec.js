import React from 'react'
import { render, cleanup } from 'react-testing-library'

import PresentationText from "./PresentationText"

afterEach(cleanup);

describe("<PresentationText />", () => {
  describe("Snapshots", () => {
    it("should match snapshot", () => {
      const { container } = render(<PresentationText />);
      expect(container).toMatchSnapshot();
    });
  })
})
