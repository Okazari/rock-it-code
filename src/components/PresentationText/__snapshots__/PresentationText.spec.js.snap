// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PresentationText /> Snapshots should match snapshot 1`] = `
<div>
  <div
    class="sc-bdVaJa khUSez"
  >
    <h1
      class="sc-bwzfXH ecxRwC"
    >
      Welcome to Rock it Code
    </h1>
    <h2
      class="sc-htpNat jTCRcv"
    >
      Simply paste your code in the editor below, press play and let it rock
    </h2>
  </div>
</div>
`;
